client/src/styles/PropertiesMap.scss

```diff
@@ -39,6 +39,43 @@
         }
       }
     }
+
+    .warning-banner {
+      position: absolute;
+      z-index: 1000;
+      top: 0;
+      margin: 1.1rem 1.1rem 0 4.5rem;
```

Comment:
I'd make the top margin piece here 1rem instead of 1.1rem — seems to make the banner align better with the zoom control on the map.